### DISCUSSION TITLE
Can we install and use loxilb in pure docker based environments ?

### DISCUSSION THREAD

**Discussion by megarajan** [loxilb-io/loxilb#873] [Q&A] [answered]
If there are any faqs/guides on using loxilb for pure docker installations , can you please point out

thanks in advance

**Comment by TrekkieCoder** [comment, marked as answer]
Hi @megarajan 

Loxilb can be used in pure docker environments. Please follow these guides :
1. [How to run with docker](https://github.com/loxilb-io/loxilbdocs/blob/main/docs/standalone.md)
2. [More commands to configure loxilb in docker mode](https://github.com/loxilb-io/loxilbdocs/blob/main/docs/cmd.md)

**Reply by megarajan** [reply]
thanks very much for the quick reply .

**Reply by megarajan** [reply]
hi , is there any page with basic information on how to test loxilb once its up and running ? 

thanks in advance

**Reply by TrekkieCoder** [reply]
There are a lot of docker based cicd scripts in the loxilb repo. For example to run a simple tcplb scenario :

```
cd cicd/tcplb
./config.sh
./validation.sh
./rmconfig.sh
```

You can dig into the scripts to check it does. Additionally, there are some demo use-cases in [killercoda](https://killercoda.com/netlox) as well but not actively maintained.

**Reply by megarajan** [reply]
thanks very much for the info.
